src/anchorfm-pupeteer/index.js

```diff
@@ -110,6 +110,14 @@ async function postEpisode(youtubeVideoInfo) {
   }
 
   async function login() {
+    if (env.ANCHOR_LOGIN) {
+      await anchorLogin();
+    } else {
+      await spotifyLogin();
+    }
```

Comment:
I changed my mind compared to the issue, let's do like this without adding a new variable:
```suggestion
    if (env.SPOTIFY_EMAIL) {
      await spotifyLogin();
    } else {
      await anchorLogin();
    }
```

Reply:
I see your point. Alright, let's keep the new env constant to handle this and eventually we can refactor this when Spotify will remove the anchor.fm signin. Could you please update the documentation in the README to add these new ENV constants?